cmd/horcrux/cmd/state.go

```diff
@@ -69,12 +72,14 @@ func showStateCmd() *cobra.Command {
 
 func setStateCmd() *cobra.Command {
 	return &cobra.Command{
-		Use:          "set [height]",
+		Use:          "set [chain-id] [height]",
```

Comment:
yes, I think it's still useful. This is used to set the last signed height to avoid double sign. It can also be done with the import state cmd to import an exact state file, but I have found this one useful separately. For example, in the case that a chain is moving during a validator migration:
- stage new horcrux validator (do not start)
- tear down old validator
- confirm missing blocks
- get latest blockchain height
- run state set cmd with that height for the chain id
- start new horcrux validator

Reply:
👏